### DISCUSSION TITLE
Score email with entire text in capital letters

### DISCUSSION THREAD

**Discussion by flaviozluca** [rspamd/rspamd#4369] [Ideas]
Idea, see attachment please.

Score email with entire text in capital letters.
[KINDLY GET BACK TO ME - 15012023 916.zip](https://github.com/rspamd/rspamd/files/10428755/KINDLY.GET.BACK.TO.ME.-.15012023.916.zip)

**Comment by fatalbanana** [comment]
Which entire text did you have in mind? There is `SUBJ_ALL_CAPS` rule which this message is hitting.

**Comment by flaviozluca** [comment]
Hi, Fatal Banana

Always you, thanks. ;)

I meant in the body text. Possible?

Tks.
